src/applications/personalization/profile/actions/personalInformation.js

```diff
@@ -101,7 +101,7 @@ export function createPersonalInfoUpdate({
       set(
         transaction,
         'data.attributes.transactionId',
-        `${fieldName}_${transaction.source_date}`,
+        `${fieldName}_${transaction?.attributes?.[fieldName]?.sourceDate}`,
```

Comment:
the structure of a successful PUT request is slightly different than the swagger docs, so they are being updated and the 'id' creation here also needed to be updated for the correct path to the timestamp property